test/test_metar.py

```diff
@@ -39,6 +39,16 @@ def test_module():
     assert hasattr(metar, "__version__")
 
 
+def test_issue114_multiplebecominggroups():
+    """multiple BECMG (becoming) groups should be possible"""
+    code = (
+        "METAR WSSS 280900Z 26009KT 180V350 0600 R20R/1900D R20C/1600D +TSRA FEW008 SCT013CB FEW015TCU 24/23 Q1010 "
+        "BECMG FM0920 TL0930 3000 TSRA "
+        "BECMG FM1000 TL1020 6000 NSW"
+    )
+    assert Metar.Metar(code).decode_completed
```

Comment:
Thanks, maybe it would be nice to test the length of `metar._trend_groups` or the contents of the trend groups.  I realize that `_` implies a private attribute.

Reply:
Thanks for your hint, I extended the test case accordingly.